docs/backend-boilerplate.md

```diff
@@ -27,7 +28,7 @@ This template features all the latest tools and best practices in the industry.
 ### Step by step
 1. Clone the repo using the command `git clone --depth=1 https://github.com/eoscostarica/backend-boilerplate.git`
 1. Access the directory using the command: `cd backend-boilerplate`
-1. Copy `.env.example` file and then update the variables according to your needs.
+1. Copy `.env.example` file and then update the variables according to your needs
```

Comment:
Maybe here instead of removing the dot, add it to the other sentences?

Reply:
My bad, good observation.